lib/perl/Genome/Model/Build/Command/Start.pm

```diff
@@ -143,6 +143,9 @@ sub create_and_start_build {
 
     my $outer_transaction = UR::Context::Transaction->begin();
     $model->build_requested(0);
+    my $anp = $model->analysis_project;
+    my $config_dir = $anp? $anp->environment_config_dir : undef;
```

Comment:
I would be happy to make that change if @jasonwalker80 thinks we're at the point where we can enforce the Analysis Project requirement!